### ISSUE TITLE
Biome-Specific Ores can't drop items with metadata.

### ISSUE THREAD

**Issue by Delocuro** [MinestrapTeam/Minestrappolation-4#16]
Since the method in the biome-specific ore classes that determines the metadata dropped when harvested has to return the metadata of the stone-type (so that the ores give the correct stone-type when harvested with Silk Touch), the metadata of dropped items can't be specified there. But the "item dropped" part of the MOres constructor doesn't support itemstacks, so the dropped item's metadata can't be specified there either. This causes problems for things such as Lapis or Coal Ore, since both of those items require specific metadata values when dropped.

Basically we either need to find some way of making the ores drop one metadata when harvested normally and a different metadata when harvested with Silk Touch, or we need to make the dropped item part of the MOres constructor somehow support metadata.
